src/cc-index.ts

```diff
@@ -29,10 +28,7 @@ export class CCIndex {
     public defaultCCIndex : string;
 
     constructor(crawlVersion? : string) {
-        if (!crawlVersion) {
```

Comment:
CC index can't access CLI to get parameters anymore but at the same time `crawlVersion` version is optional. 

So this will break if we don't provide `crawlVersion` --> it shouldn't be optional anymore.